grammar.js

```diff
@@ -29,7 +29,10 @@ const PRECEDENCE = {
 
 module.exports = grammar({
   name: 'move',
-  extras: $ => [$._whitespace, $.line_comment, $.block_comment, $.newline, $.annotation],
+  //extras: $ => [$._whitespace, $.line_comment, $.block_comment, $.newline, $.annotation],
+  extras: $ => [$.line_comment, $.block_comment, $.annotation, /\s/],
```

Comment:
@damirka do you still need empty lines for the formatter?

Reply:
I do need!